docs/docs/manual/columnediting.md

```diff
@@ -0,0 +1,128 @@
+*   and **arg4** is which value in that column to import (most likely 0).
+
+Learn more about [cross()](expressions#cross) and other GREL functions to use in this window on the [Expressions](expressions) page. 
+
+Some of the other most common ways to add a new column based on an existing one are separate functions, and are explained below.
+
+
+## Add column by fetching URLs
+
+Through the "Add column by fetching URLs" function, OpenRefine supports the ability to fetch HTML or data from web pages or services. In this operation you will be taking strings from your selected column and inserting them into URL strings. This presumes your chosen column contains parts of paths to valid HTML pages or files online. 
+
+If you have a column of URLs and watch to fetch the information that they point to, you can simply run the expression as `value`. If your column has, for example, unique identifiers for Wikidata entities (numerical values starting with Q), you can download the JSON-formatted metadata about each entity with
+
+```“https://www.wikidata.org/wiki/Special:EntityData/” + value + “.json”```
+
+or whatever metadata format you prefer. [Information about these Wikidata options can be found here](https://www.wikidata.org/wiki/Wikidata:Data_access).
+
+This service is more useful when getting metadata files instead of HTML files, but you may wish to work with a page’s entire HTML contents and then parse out information from that. Be cautioned that the fetching process can take quite some time and that servers may not want to fulfill hundreds or thousands of page requests in seconds. 
+
+Fetching allows you to set a “throttle delay” which determines the amount of time between requests. The default is 5 seconds per row in your dataset (5000 milliseconds), so you can estimate how long it will take to work through the entire column using that number. We recommend leaving this value at 5000 or greater. 
+
+![A screenshot of the settings window for fetching URLs.](/img/fetchingURLs.png)
+
+Note the following:
+
+* Many systems prevent you from making too many requests per second. To avoid this problem, set the throttle delay, which tells OpenRefine to wait the specified number of milliseconds between URL requests.
+* Before pressing OK, copy/paste a URL or two from the right column in the dialog and test them in another browser tab to make sure they work.
+* In some situations you may need to set[ HTTP request headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers). To set these, click the small “Show” button next to "HTTP headers to be used when fetching URLs" in the settings window. You can set the following request headers:
+  * [User-Agent](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent)
+  * [Accept](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept)
+  * [Authorization](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization)
```

Comment:
We should warn users that the authorization credentials get logged in the operation history in plain text, representing a security concern that they need to be aware of.